Session.js

```diff
@@ -49,21 +49,33 @@ export default class Session {
     await this.refresh();
   }
 
+  /**
+   * Registers a callback that is executed when an event listener fires.
```

Comment:
I think we should stick with `handler` as per the params instead of introducing `callback` which also has a different connotation.

Reply:
Agreed and addressed here: https://github.com/digitalbazaar/bedrock-web-session/pull/4/commits/bbbcf89c91e9364ea66cbcffae873f8af48b516a